packages/components/pie-button/tsconfig.json

```diff
@@ -1,8 +1,11 @@
 {
-  "extends": "../../../configs/pie-components-config/tsconfig.json",
```

Comment:
Its smaller PRs but any reason we can't change each component (or smaller groups) in separate PRs?

Reply:
No reason, but I do think it's unnecessary personally. I'm happy to do that if others don't have the same risk tolerance, but this change won't affect consumers, only PIE

Reply:
It's less about consumer risk and more about tracking regressions. I am happy to go with the majority opinion

Reply:
Sure thing.

We're not even using the shared config in the aperture apps, so there's no risk there for now.

I did do a quick test locally to extend the aperture apps tsconfigs and didn't observe and TS / IDE issues.

Also can't spot anything within PIE 👍🏼.